### DISCUSSION TITLE
Addition of Depth Value alongside is_full_depth

### DISCUSSION THREAD

**Discussion by danner26** [netbox-community/netbox#11960] [Ideas]
Hello all,

Do you see any value in adding an optional field to track device `depth` alongside the `is_full_depth` boolean field? While I do not have a specific use case for this myself, I've noticed an uptick of contributors in the [devicetype-library](https://github.com/netbox-community/devicetype-library) adding comments/reference to a devices specific depth. We have not been asked to implement this directly, however, I noticed it is a common occurrence; thus this discussion is born.

Do you find this data point to be useful for tracking within NetBox? Can you think of, or do you currently have a specific use case that would benefit from tracking this information? If so, please leave a thumbs up, comment or something similar to indicate interest. Thanks!

**Comment by a084ed22** [comment]
is_full_depth is meaningful when there is a standardized rack size so that the device depth, relative to that of the rack, can make a given unit available on the opposite side the device is mounted on or not.

In my environment we have shallow "networking" racks and deeper "server" racks that may still contain networking equipment a depth indication would be more useful.

**Reply by danner26** [reply]
Yup that is a good use case, my MSP utilizes NetBox to document client racks. At this moment we don't specifically need the depth field but I do see where it would be useful when you end up racking hardware on the same RU front and back
